internal/impl/kube.go

```diff
@@ -489,6 +495,82 @@ func GenerateYAMLs(image string, app *protos.AppConfig, depId string, cfg *KubeC
 	return nil
 }
 
+// header returns the informational header at the top of a generated YAML file.
+func header(app *protos.AppConfig, cfg *KubeConfig, depId, filename string) (string, error) {
+	header := template.Must(template.New("header").Parse(`# This file was generated by "weaver kube" version {{.ToolVersion}} for the following
+# application:
+#
+#     app: {{.App}}
+#     version: {{.Version}}
+#     components:
+      {{- range .Components}}
+#     - {{.}}
+      {{- end}}
+#     listeners:
+      {{- range .Listeners}}
+#     - {{.}}
+      {{- end}}
+#
+# This file contains the following resources:
+#
+#     1. A Deployment for every group of components.
```

Comment:
would it be possible to list how these groups of components look like?

Reply:
Good idea, done!